src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -33,38 +94,226 @@ const ContactUs = () => {
+                      <Text>BusinessCards</Text>
+                    </HStack>
+                    <HStack marginBottom={'17%'}>
+                      <NumberInput
+                        size="sm"
+                        maxW={20}
+                        defaultValue={0}
+                        min={0}
+                        onChange={e => handleNumInputChange(e, 4)}
+                      >
+                        <NumberInputField />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                      <Text>Stickers</Text>
+                    </HStack>
+                  </Stack>
+                  <Stack margin={'5%'}></Stack>
+                  <Stack>
+                    <HStack marginBottom={'17%'}>
+                      <NumberInput
+                        size="sm"
+                        maxW={20}
+                        defaultValue={0}
+                        min={0}
+                        onChange={e => handleNumInputChange(e, 5)}
+                      >
+                        <NumberInputField />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                      <Text>Bin Decals</Text>
+                    </HStack>
+                    <HStack marginBottom={'17%'}>
+                      <NumberInput
+                        size="sm"
+                        maxW={20}
+                        defaultValue={0}
+                        min={0}
+                        onChange={e => handleNumInputChange(e, 6)}
+                      >
+                        <NumberInputField />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                      <Text>DS Decals</Text>
+                    </HStack>
+                  </Stack>
+                </Box>
+              </Card>
+              <Box margin={'5%'}>
+                <Textarea
+                  width={'90%'}
+                  h={'150px'}
+                  value={text}
+                  placeholder="Notes..."
+                  onChange={textInputHandler}
+                />
+              </Box>
+              <Spacer p={'6%'}></Spacer>
+              <HStack justify="flex-end" marginEnd={'5%'}>
+                <Button colorScheme="teal" variant="outline" width="130px">
+                  Cancel
+                  <Link to="/BusinessDashboard"></Link>
+                </Button>
+                <Button {...buttonStyles} disabled={isFormValid} width="130px" onClick={SubmitForm}>
```

Comment:
the button is still clickable even when form is not valid. this is because the prop should be isDisabled not disabled.